vendor.yaml

```diff
@@ -183,6 +183,14 @@ libs:
     npmDir: kaitai-struct
     files: [LICENSE, KaitaiStream.js]
 
+  js-yaml:
+    source: https://github.com/nodeca/js-yaml
+    licenseName: MIT
+    licenseUrl: https://github.com/nodeca/js-yaml/blob/master/LICENSE
+    npmDir: js-yaml
+    files: [LICENSE, dist/js-yaml.min.js]
+
+  # FIXME: remove when no longer referenced by src/KaitaiSandbox.ts and src/worker/ImportLoader.ts
```

Comment:
Similar question: given that we'll close #165 by merging this PR, shall we create follow-up issue to track these separately? Otherwise, we'll likely forget.

Reply:
See https://github.com/kaitai-io/kaitai_struct_webide/issues/168